src/cosmosdb-preview/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.4.0
```

Comment:
Did you forget the version in `setup.py`?

Reply:
I did. Thanks! :) Fixed now